README.md

```diff
@@ -50,7 +50,7 @@ fcrepo:
   servicePort: 8080
   postgresql:
     enabled: false
-    postgresqlHost: someurl.rds.amazonaws.com
+    postgresqlHost: someurl.rds.amazonaws.com # Optional, only specify if not using internal chart defined postgersql
```

Comment:
maybe move this to the top level and make it `externalPostgresqlHost`?

i've varied on my practice here over time, but i think what i've learned is: don't namespace Values under a chart dependency unless you intend to pass them through to the subchart.

Reply:
the weird thing to me here is that you setting username and password inside the block either way (which makes sense as they are the postgres terms either way. do you pull them all out a level (which I think you just undid ;-) or mix and match? it doesn't feel "right" to me in any of these ways yet.